src/gx.rs

```diff
@@ -3,6 +3,7 @@
 //! This module implements a safe wrapper around the graphics functions found in ``gx.h``.
 
 use core::ffi::c_void;
+use crate::ffi;
 use ogc_sys::{Mtx as Mtx34, Mtx44};
```

Comment:
Please keep all imports off the same crate in the same use. EX instead of `use ogc_sys`, use the crate:ffi` for everything

Reply:
Okay, I've replaced all uses of `ogc_sys` with `ffi`